runtime/src/accounts_db.rs

```diff
@@ -3724,23 +3724,30 @@ impl AccountsDb {
         })
     }
 
+    /// 'accounts' are about to be appended to an ancient append vec. That ancient append vec may already have some accounts.
     /// unref each account in 'accounts' that already exists in 'ancient_store'
```

Comment:
`ancient_store` is no longer an argument here, can you update this comment?

Reply:
added in follow on pr